backend/src/TodosDatabase.ts

```diff
@@ -0,0 +1,64 @@
+import { TodoDto } from './dto/todoDto';
+import { ConnectionService } from './app.connectionService';
+
+export default class TodosDatabase {
+  private static connectionService = new ConnectionService();
+
+  static async createTodo(todo: TodoDto) {
+    const newTodoOrder = await this.getNextTodoMaxOrder();
```

Comment:
computing the next id should be part of the service. This object should only be the pure interface that works with the Todos table.
Same throughout this file.